packages/builder/src/components/settings/UpdateAPIKeyModal.svelte

```diff
@@ -0,0 +1,37 @@
+<script>
+  import { ModalContent, Body, notifications } from "@budibase/bbui"
+  import { auth } from "stores/portal"
+  import { onMount } from "svelte"
+  import CopyInput from "components/common/inputs/CopyInput.svelte"
+
+  let apiKey = null
+
+  async function generateAPIKey() {
+    try {
+      apiKey = await auth.generateAPIKey()
+      notifications.success("New API key generated.")
```

Comment:
NAB: If you remove the full stop it will make me smile 🙏

Reply:
Would it be an idea to adjust the notification to strip full stops if they are at the end? Will fix it in code too but would be good if component made sure that a convention was always followed too!